Blogplace.Web/Configuration/EmailOptions.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Blogplace.Web.Configuration;
+
+public class EmailOptions
+{
+    public string Host { get; set; } = "localhost";
+    public int Port { get; set; } = 587;
```

Comment:
tu w sumie port mógłby być pusty, wtedy ustawi 0 

jak będą lecieć błędy to łatwo sie domyślimy że nie ładuje z konfiguracji